package.json

```diff
@@ -30,28 +30,28 @@
   },
   "scripts": {
```

Comment:
While we're here we should fix the warnings yarn throws up:

```
warning " > @emotion/styled@10.0.12" has unmet peer dependency "@emotion/core@^10.0.0".
warning "@emotion/styled > @emotion/styled-base@10.0.13" has unmet peer dependency "@emotion/core@^10.0.0".
warning " > react-emotion@9.0.2" has incorrect peer dependency "emotion@^9.0.2".
```

We can remove react-emotion as we don't use that version anymore, and we can add @emotion/core to the root package.json as it's a peerDep.

Reply:
👍